storybook/qmlTests/src/TextUtils.cpp

```diff
@@ -0,0 +1,25 @@
+#include "TextUtils.h"
+
+#include <QQmlEngine>
+#include <QTextDocumentFragment>
+
+TextUtils::TextUtils(QObject *parent) :
+    QObject(parent)
+{
+
+}
+
+QObject *TextUtils::qmlInstance(QQmlEngine *engine, QJSEngine *scriptEngine)
+{
+    Q_UNUSED(engine)
+    Q_UNUSED(scriptEngine)
+
+    static TextUtils instance;
+    QQmlEngine::setObjectOwnership(&instance, QQmlEngine::CppOwnership);
+
+    return &instance;
+}
+
+QString TextUtils::htmlToPlainText(const QString &html) {
+    return QTextDocumentFragment::fromHtml( html ).toPlainText().toUtf8();
```

Comment:
`toPlainText()` already returns a `QString`; converting it with `.toUtf8()` is wrong

Reply:
Ohh, you're right! Good catch!